packages/vega-voronoi/src/Voronoi.js

```diff
@@ -22,25 +22,23 @@ Voronoi.Definition = {
 
 var prototype = inherits(Voronoi, Transform);
 
-var defaultExtent = [[-1e5, -1e5], [1e5, 1e5]];
+var defaultExtent = [-1e5, -1e5, 1e5, 1e5];
 
 prototype.transform = function(_, pulse) {
   var as = _.as || 'path',
       data = pulse.source,
-      diagram, polygons, i, n;
+      delaunay, extent, voronoi, polygon, i, n;
 
   // configure and construct voronoi diagram
-  diagram = voronoi().x(_.x).y(_.y);
-  if (_.size) diagram.size(_.size);
-  else diagram.extent(_.extent || defaultExtent);
-
-  this.value = (diagram = diagram(data));
+  delaunay = Delaunay.from(data, _.x, _.y);
+  extent = _.size ? [0, 0 , ..._.size] : _.extent ? [..._.extent[0], _.extent[1][0]] : defaultExtent;
```

Comment:
I think you’ll want `[..._.extent[0], ..._.extent[1]]` here?

Reply:
Yes. Not sure what my past me was thinking here.